src/ui/public/vislib/visualizations/point_series/column_chart.js

```diff
@@ -47,19 +47,21 @@ export default function ColumnChartFactory(Private) {
         return 'series ' + i;
       });
 
+      const layers = data.values;
+
       const bars = layer.selectAll('rect')
-        .data(layers);
+      .data(layers);
```

Comment:
bad naming .... probably someone has taught of bar chart series as layers (make sense in a way when you are stacking them ...)